### PR TITLE
added my profile `@theabhayprajapati`

### PR DESCRIPTION
**Pull request by theabhayprajapati**

### CODE REVIEW

**Review by theabhayprajapati — COMMENTED** [review]
added my profile

File: public/data/abhayprajapati.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Abhay Prajapati",
+    "type": "personal",
+    "bio": "frontend dev",
+    "avatar": "https://github.com/theabhayprajapati.png",
+    "links": [
+      {
+        "name": "My Personal Page",
+        "url": "http://abhayprajapati.me",
```
Comment (kumarsonsoff3) [inline public/data/abhayprajapati.json:9]: Your link is using a secure protocol! 

```suggestion
        "url": "https://abhayprajapati.me",

```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Few Fixes required! 🧰 

- Change your filename to `theabhayprajapati.json`
- Update the link as suggested
- Remove the empty space from the end! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by theabhayprajapati — COMMENTED** [review]
checked the changes

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
**Still you need to -**

- Change your filename to from `abhayprajapati.json` to `theabhayprajapati.json`
- Edit the description of your PR. ( Description could not be empty )

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by theabhayprajapati — COMMENTED** [review]
updated the file name to usename

**Review by EmmaDawsonDev — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

**Pull request merged**